CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+## 0.4.19
+
+**releases 2022-06-14**
+
+### Added
+
+* Release on tag creation  [#172](https://github.com/freiheit-com/kuberpult/pull/172)
+* Increase default loadbalancer timeout to 300 [#183](https://github.com/freiheit-com/kuberpult/pull/183)
+
+### Fixed
+
+* Increase Kuberpult's memory limit & request [#182](https://github.com/freiheit-com/kuberpult/pull/182)
+
```

Comment:
Changelog contains all significant changes since v0.4.18: https://github.com/freiheit-com/kuberpult/pulls?q=is%3Apr+is%3Aclosed+sort%3Aupdated-desc :heavy_check_mark: